src/Oro/Bundle/DataGridBundle/Resources/public/js/datagrid/page-size.js

```diff
@@ -110,7 +110,7 @@ define([
                 _.bind(
                     function(item) {
                         return item.size === undefined ?
-                            this.collection.state.pageSize === item : this.collection.state.pageSize === item.size;
+                            this.collection.state.pageSize == item : this.collection.state.pageSize == item.size;
                     },
```

Comment:
@Grygir So what kind of fix will be acceptable? Explicit string to number type conversion here and then strict equality checking?